### DISCUSSION TITLE
(1364, "Field 'locked' doesn't have a default value")

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#7969] [Q&A]
I suddenly start to have this error, is this related to Inventree or it's the database?

**Comment by SchrodingersGat** [comment]
Can you please provide the complete error message

**Reply by jesseyu1984918** [reply]
After updating to latest docker image, I thought it's resolved and adding on more part works. but then I don't see the part and this lock error starts to occur

Error object (67)
[History](https://inventree.sdprd.oks.drekar.qualcomm.com/admin/error_report/error/67/history/)

Type:
IntegrityError
Path:
/api/part/
Info:
(1364, "Field 'locked' doesn't have a default value")
When:
Aug. 23, 2024, 3:09 a.m.
Data:
Traceback (most recent call last):

File "/root/.local/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
return self.cursor.execute(sql, params)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

File "/root/.local/lib/python3.11/site-packages/django/db/backends/mysql/base.py", line 75, in execute
return self.cursor.execute(query, args)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

File "/root/.local/lib/python3.11/site-packages/MySQLdb/cursors.py", line 179, in execute
res = self._query(mogrified_query)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

File "/root/.local/lib/python3.11/site-packages/MySQLdb/cursors.py", line 330, in _query
db.query(q)

File "/root/.local/lib/python3.11/site-packages/MySQLdb/connections.py", line 261, in query
_mysql.connection.query(self, query)

MySQLdb.IntegrityError: (1364, "Field 'locked' doesn't have a default value")

**Reply by jesseyu1984918** [reply]
In the part_part table, there is a column called locked and somehow it can't be created

**Reply by SchrodingersGat** [reply]
After you pulled down the latest image, did you run `invoke update`?

**Reply by jesseyu1984918** [reply]
> After you pulled down the latest image, did you run `invoke update`?
Yes, and it's sporatically showing the error..

**Comment by SchrodingersGat** [comment]
Also please provide the version information. Go to "about InvenTree" and "copy version information"
